### DISCUSSION TITLE
FEATURE REQUEST. Player count display

### DISCUSSION THREAD

**Discussion by supavgx** [aldostools/webMAN-MOD#539] [Ideas]
As of right now when playing with a second player I Google the game in hope it's two player.. is it possible to have a display of how many players the game allows. Either while scrolling through games or after a game has be been loaded.. I guess one way to do it it is by pulling information from SFO and as of now while viewing SFO files in param.sfo editor doesn't shows this but maybe it was can be added to it?

**Comment by aldostools** [comment]
As far as I know the PARAM.SFO doesn't include information about how many players a game allows.

You can use the /dev_hdd0/wm_custom.xml to create manually a customized menu including that information.

Or you can group the ISOs in sub-folders. However, games in folder format cannot be placed in subfolders.

Another alternative is to use the 4 game profiles to group by number of players (GAMES_1, PS3ISO_1, GAMES_2, PS3ISO_2, etc.) 
e.g. 1=for single player, 2=for dual players, 3=for 3-4 players, 4=for 5+ players
Reserve GAMES, PS3ISO (without sufix) for your favorite games that you want to be displayed on all profiles.
This method requires that you refresh XMB when you change from profile.

Another method is editing the Title in PARAM.SFO, but I do not recommend to do it because the name is exposed publicly when you play online on PSN.
